lib/SDIR/Translate/translateToSDFG.cpp

```diff
@@ -0,0 +1,1597 @@
+#include "SDIR/Translate/Translation.h"
+#include "mlir/Dialect/StandardOps/IR/Ops.h"
+#include "mlir/IR/AsmState.h"
+
+using namespace mlir;
+using namespace sdir;
+
+//===----------------------------------------------------------------------===//
+// TranslateToSDFG
+//===----------------------------------------------------------------------===//
+
+// NOTE: Prefer calling the specific translating functions instead
+LogicalResult translateToSDFG(Operation &op, JsonEmitter &jemit) {
```

Comment:
This looks a bit ugly. Is there a reason a polymorphic `.translateToSDFG` isn't used?

Reply:
I agree and I've tried using a overloaded function.
I don't remember why it wasn't working. But I would suggest removing this function entirely since calling the specific functions makes it easier to follow the code